cocos/renderer/backend/opengl/TextureGL.cpp

```diff
@@ -22,10 +22,19 @@ namespace
         return ret;
     }
     
-    GLint toGLMinFilter(SamplerFilter minFilter, SamplerFilter mipmapFilter, bool mipmapEnabled)
+    GLint toGLMinFilter(SamplerFilter minFilter, SamplerFilter mipmapFilter, bool mipmapEnabled, bool bPow2)
     {
         if (mipmapEnabled)
         {
+            if(!bPow2)
+            {
+                cocos2d::log("Disable mipmap since non-power-of-two texture occur in %s %s %d", __FILE__, __FUNCTION__, __LINE__);
```

Comment:
It is better to say that what filter is used instead.

Reply:
Done!